less/input-groups.less

```diff
@@ -126,6 +126,15 @@
   vertical-align: middle; // Match the inputs
 }
 
+.input-group-currency-select-btn {
```

Comment:
Is this the class we need to add to the amount currency select? If yes, do we have a corresponding PR. Also, as it's specific to _amount-currency-select_, couldn't the name reflect that? And potentially also the file?

Reply:
I wanted to make changes in bootstrap before making a PR for component, will do a PR for the component as well then ;)
did not understand what do you mean by the file reflecting the name?

Reply:
Do amount currency selects have any other custom styles? If yes, where are they? If no, I think it'd be more obvious and clear if these styles were in a file called `amount-currency-select.less` etc.

Reply:
taking it out the `input-group.less` and creating a separate less?